### DISCUSSION TITLE
Sharing setup.toml files

### DISCUSSION THREAD

**Discussion by mattblovell** [mattblovell/kodi_panel#38] [General]
It might be nice for people to share their setup.toml files, once one's customizations are in-place and working as desired.  There may be ideas regarding different layouts, conditional display, or even fun font choices that others may want to try!

@nico1080 has suggested several ideas that enhance the TOML file's capabilities, particularly for video screens.  Even though I can't necessarily use those directly, it might be interesting to see those ideas "in action".

Short excerpts could just be shared here.  If there are more elaborate changes that someone would like to share, we can add to the example_setups/ directory.

**Comment by nico1080** [comment]
My `setup.toml` file is within my fork, so every body can just use it.

In the next few days, I will rework my fork to benefit all latest updates of kodi panel (right now I am about version 1.26) and so that the main script to have no difference with @mattblovell master branch. and move all my special treatment to separate script.

I also have to work my layouts to make them "presentable"  and post pictures of it.

Maybe making a list of all user specific treatment woulld help new user to undetstand all the possibilty. (Maybe open a wiki?)
In my next commit I will do a one commit per item these commit could be used to make this list

**Comment by nico1080** [comment]
I posted a comment on the coreelec discussion, hope it will make more people come:
https://discourse.coreelec.org/t/graphical-front-panel-display/12932/208

![IMG_20210202_195310](https://user-images.githubusercontent.com/5569758/106653176-bc162600-6596-11eb-9883-cf1614417f74.jpg)
![IMG_20210202_195351](https://user-images.githubusercontent.com/5569758/106653209-c801e800-6596-11eb-9430-f9222316afaa.jpg)
![IMG_20210202_195429](https://user-images.githubusercontent.com/5569758/106653212-c89a7e80-6596-11eb-826c-14c0d90feaf6.jpg)
![IMG_20210202_195531](https://user-images.githubusercontent.com/5569758/106653213-c89a7e80-6596-11eb-8156-414138d11e36.jpg)
![IMG_20210202_195719](https://user-images.githubusercontent.com/5569758/106653215-c9331500-6596-11eb-8149-557fef82dc05.jpg)
![IMG_20210202_195048](https://user-images.githubusercontent.com/5569758/106653216-c9331500-6596-11eb-85ac-1d570a3b4daf.jpg)
![IMG_20210202_195130](https://user-images.githubusercontent.com/5569758/106653218-c9331500-6596-11eb-908d-f90192f5e163.jpg)
![IMG_20210202_195152](https://user-images.githubusercontent.com/5569758/106653221-c9cbab80-6596-11eb-824e-23e35f6c5de6.jpg)
![IMG_20210202_195222](https://user-images.githubusercontent.com/5569758/106653222-c9cbab80-6596-11eb-90f2-1eeae3f43892.jpg)
